index.html

```diff
@@ -157,35 +157,25 @@ <h3>Full Name</h3>
             <div class="custom-card-wrapper col-12 col-md-6 col-lg-4">
               <div class="custom-card">
                 <div class="avatar-wrapper">
-                  <img src="img/placeholder.jpeg" alt="" class="avatar" />
+                  <img src="img/aog.jpg" alt="" class="avatar" />
                 </div>
                 <div>
-                  <h3>Full Name</h3>
+                  <h3>Ghita Alexandru Ovidiu</h3>
                   <p>
-                    Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                    Molestiae omnis nobis quo dicta culpa eum exercitationem
-                    provident ex iure pariatur? Tempora sunt accusamus tempore
-                    modi.
+                    Hey, im an ex-medical engineer witch has gone through IT school web development program, in order to change my carrer path.
                   </p>
                   <p>
-                    Quidem, corrupti perspiciatis aliquid dicta cum odio
-                    sapiente tempora quas incidunt adipisci alias praesentium et
-                    excepturi error maxime aspernatur sint qui doloremque
-                    distinctio. Ex, unde.
+                   I'm a team player, but can also work alone just fine, i think of myself as a problem-solver, i always give 110% in order to become better and faster in resolving issues.
```

Comment:
`as a problem-solver,` either put a full stop here, or an `and` before the next sentence.